### PR TITLE
Cogwerks Donation PR: Donate to Cogwerks 2day || Headmin Campaign Bribe chemicals

### PR DESCRIPTION
**Pull request by Iamgoofball**
This PR was coded to get Cogwerks at Goonstation 15 bucks on his patreon. Support Cogwerks, the genius behind Goonchem at goon(The actually good version that goon has, not my shitty remake), Cogmap, and Cogmap 2.

https://www.patreon.com/cogwerks

Seriously. Go throw some money at this man. He's a good person. Give him your money. Do it.

The first 3 chemicals are Sarin, Zyklon B, and VX Nerve Agent. They have fun recipes I'll be filling in shortly. All 3 are potent toxins on their own, each more and more complex and tough to make, but getting better and better as they go along.

The next 2 chemicals were part of my headmin campaign bribes for nominations. Fluoroantimonic Acid and Hunchback are the 2 chemicals.

Fluoroantimonic Acid is a very very powerful acid, that takes CLF3, Facid, and Sulf acid to make. It is obnoxious to make because it takes 10 of each, but only produces 10 units. A total of 30 to 10 unit ratio. Mass production takes time. I might lower this value even more to 5 units per 30 total.

Hunchback is a ghetto calomel. It removes reagents at 6 a tick, but will do 4 toxin damage a tick until you are at 0 health, and metabolizes slower that Calomel. Calomel does 2.5 removal a tick, with 2.5 toxin damage until you're at health 20. Hunchback is intended for bartenders to whip up in small batches and keep on hand for emergency bar poison control. Don't dose people with a shitton, use small doses. Unless you're a traitor. In which case, go ahead. 

I'll add all recipes to this post SOON(tm)

### CODE REVIEW

**Comment by cyfause** [comment]
no

**Comment by Iamgoofball** [comment]
@cyfause explain

File: code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm [hunk 1]
```diff
@@ -609,6 +609,80 @@
...
+		var/turf/pos = get_turf(M)
+		pos.add_vomit_floor(M)
+		playsound(pos, 'sound/effects/splat.ogg', 50, 1)
+	M.adjustStaminaLoss(3)
+	if(prob(25))
+		M.adjustBrainLoss(5)
+		M.adjustBruteLoss(3)
+	..()
+
+/datum/reagent/toxin/zyklon_b
```
Comment (RemieRichards) [inline code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm]: Ahahaha... I don't think this can slide.

Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm]: What's the problem here?

**Comment by xxalpha** [comment]
>give money to closed source project

nah

**Comment by Iamgoofball** [comment]
don't you mean
>give money to cool dude who happens to also work on a closed source project

**Comment by Miauw62** [comment]
goonchem is only good because of secret club bullshit tho

File: code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm (hunk 1 above)
Comment (Miauw62) [inline code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm]: >holocaust gas
>whats the problem here

Comment (Delimusca) [inline code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm]: could use the english name, cyclone B. or the lesser known precursor, zyklon A.

**Comment by SconesC** [comment]
"oyyyy veyyyyy nevah let the goyim make a joke"

**Comment by Fox-McCloud** [comment]
Zyklon B is just cyanide.

**Comment by Iamgoofball** [comment]
nah, it's technically hydrogen cyanide

before the nazis came and ruined it it was actually a commercial pesticide

**Comment by Fox-McCloud** [comment]
whenever refers to cyanide generically, they're nearly always referring to hydrogen cyanide. For example, the "cyanide" that's already a recipe is hydrogen cyanide: https://en.wikipedia.org/wiki/Hydrogen_cyanide#Production_and_synthesis with oil acting as a replacement for methane.

**Comment by Strathcona** [comment]
There is absolutely zero reason, given all the other fictional or historically "okay" chemical names to pick one with such a sordid history. Pick a different name and avoid the drama train.

**Comment by SconesC** [comment]
Call it Zyklon L

**Comment by Dorsisdwarf** [comment]
Remove Zyklon B, we already have cyanide

File: code/modules/reagents/Chemistry-Recipes/Others.dm [hunk 2]
```diff
@@ -351,4 +351,18 @@
...
+	result_amount = 3
+
+/datum/chemical_reaction/phosphorus_trichloride
+	name = "Phosphorus Trichloride"
+	id = "phosphorus_trichloride"
+	result = "phosphorus_trichloride"
+	required_reagents = list("phosphorus" = 1, "chlorine" = 3)
+	result_amount = 4
+
+/datum/chemical_reaction/methyl_phosphonous_dichloride
```
Comment (Profakos) [inline code/modules/reagents/Chemistry-Recipes/Others.dm]: This recipe and the one above seem to refer to a chemical not in the code in any way

Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Recipes/Others.dm]: i fucked up and forgot to commit those, whoops

**Comment by Aranclanos** [comment]
shouldn't you ask for permission to start shilling patreons here

**Comment by Iamgoofball** [comment]
will revisit after Black Ops is done

**Comment by tuypo1** [comment]
@Strathcona theres no reason not to pick it either if you avoid the past your doomed the repeat it

**Comment by Strathcona** [comment]
Except for avoiding things like attracting racists or neo-nazis or being associated with them.

**Comment by Iamgoofball** [comment]
i dont know who either of you are

**Comment by Miauw62** [comment]
lmao dont listen to tuypo, he got banned from a forum i frequent for being an asshole and going "fuk libruls" all the time.

**Comment by Iamgoofball** [comment]
what forum miauw

**Comment by Strathcona** [comment]
No, I think for the most part any jerk who would abuse things is already part of the player base. The trick is to not give them tools to do so.

**Comment by Aranclanos** [comment]
oy shut up faget

**Pull request closed without merging**